src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -247,6 +272,44 @@ public synchronized ProcessWrapper launchTranscode(
 		return pw;
 	}
 
+	/**
```

Comment:
@Sami32 "when i do the changes asked" you didn't do the changes asked. I said to add `@override` *if* the methods are different otherwise we shouldn't need to add them, and they are not different :)